batch/python/resample.py

```diff
@@ -160,15 +161,92 @@ def download_tiles(args: Tuple[str, str]) -> str:
     source_tile_uri, dest_dir = args
 
     local_src_file_path = os.path.join(dest_dir, os.path.basename(source_tile_uri))
-    bucket, source_key = get_s3_path_parts(source_tile_uri)
 
-    s3_client = get_s3_client()
-    print(f"Downloading {source_tile_uri} to {local_src_file_path}")
-    s3_client.download_file(bucket, source_key, local_src_file_path)
+    # TODO: Skip download when possible. Use checksum?
+    if os.path.isfile(local_src_file_path):
+        print(f"Local file {local_src_file_path} already exists, skipping download")
+    else:
+        bucket, source_key = get_s3_path_parts(source_tile_uri)
+        s3_client = get_s3_client()
+        print(f"Downloading {source_tile_uri} to {local_src_file_path}")
+        s3_client.download_file(bucket, source_key, local_src_file_path)
 
     return local_src_file_path
 
 
+def exists_in_s3(target_bucket, target_key):
```

Comment:
should this go [here](https://github.com/wri/gfw-data-api/blob/master/app/utils/aws.py)?

Reply:
As things are we don't copy all the data API code into the Batch containers, just the stuff under /batch
But I agree it is general enough that it should be factored-out. Will do so.